packages/@uppy/companion/src/server/Uploader.js

```diff
@@ -480,7 +480,8 @@ class Uploader {
       if (error) {
         this.emitError(error)
       } else {
-        this.emitSuccess(null, {
+        const url = data && data.Location ? data.Location : null;
```

Comment:
```suggestion
        const url = data && data.Location ? data.Location : null
```

Reply:
Added suggestion for lint fixing — we don’t use semicolons in Uppy project.

Reply:
Thanks! :D